### PR TITLE
Multi processing

### PR DESCRIPTION
**Pull request by arikalon1**

### CODE REVIEW

File: src/robusta/core/sinks/robusta/dal/supabase_dal.py [hunk 1]
```diff
@@ -129,7 +129,7 @@ def to_service(self, service: ServiceInfo) -> Dict[Any, Any]:
             "account_id": self.account_id,
             "deleted": service.deleted,
             "service_key": service.get_service_key(),
-            "config": service.service_config.dict(),
+            "config": service.service_config.dict() if service.service_config else None,
```
Comment (aantn) [inline src/robusta/core/sinks/robusta/dal/supabase_dal.py:132]: Does this fix a bug unrelated to the rest of the changes?

File: src/robusta/integrations/prometheus/trigger.py [hunk 2]
```diff
@@ -87,22 +106,23 @@ def __find_node_by_ip(cls, ip) -> Optional[Node]:
                 return node
         return None
 
-    @classmethod
-    def __load_node(cls, alert: PrometheusAlert, node_name: str) -> Optional[Node]:
+    @staticmethod
```
Comment (aantn) [inline src/robusta/integrations/prometheus/trigger.py:109]: Any reason for changing this from a classmethod to staticmethod? It doesn't actually matter much, but you're calling another method on the same class (`__find_node_by_ip`). By keeping it as a class method and using `cls` the code will work even if someone changes the name of the class in the future.

Comment (aantn) [reply src/robusta/integrations/prometheus/trigger.py:109]: Same question applies to other methods below.

**Review by aantn — APPROVED** [review]
Looks good. Left some comments.

File: src/robusta/core/sinks/robusta/dal/supabase_dal.py (hunk 1 above)
Comment (arikalon1) [reply src/robusta/core/sinks/robusta/dal/supabase_dal.py:132]: yes

File: src/robusta/integrations/prometheus/trigger.py (hunk 2 above)
Comment (arikalon1) [reply src/robusta/integrations/prometheus/trigger.py:109]: I'll change it back

**Pull request merged**